.github/workflows/tests.yml

```diff
@@ -37,26 +37,30 @@ jobs:
       - name: Lint ✨
         run: npm run test:lint
 
-  Build:
+  Release:
     needs: [Test, Lint]
-    if: github.ref == 'refs/heads/master'
+    if: |
+      github.ref == 'refs/heads/master' &&
+      github.event.repository.fork == false
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Install Dependencies
         run: npm ci
       - name: Build 🗜️
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           npm run build
           if ! git diff --quiet; then
             git config --global user.email "<>"
             git config --global user.name "MarkedJS bot"
             git commit -am "🗜️ build [skip ci]"
-            git push "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/markedjs/marked.git" HEAD:master
           fi
+      - name: Release 🎉
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx sematic-release
```

Comment:
Where is this coming from? I don't see a package named [semantic-release](https://www.npmjs.com/package/sematic-release) in npm.

Reply:
typo

Reply:
Since `semantic-release` isn't defined in package.json, this is going to install the latest.

I think we should use a specific version of semantic-release and put it in package.json devDependencies like the rest.

Reply:
It is in package.json. This will run the version that is in package.json

Reply:
Oh yes you're right, I see it on line 60 🤦‍♂️